gradoop-accumulo/src/main/java/org/gradoop/common/utils/Ins.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradoop.common.utils;
+
+import java.util.ArrayList;
+import java.util.function.Function;
+
+/**
+ * instance operation for java8
+ */
+public final class Ins {
```

Comment:
This class name is not expressive. It should be clear by the class name what it is meant for. I'm even not sure whether we really need this class.

Reply:
Sorry, I'll remove it. 

BTW, this class means `InstanceOperatorUtils`, which provides a sets of instance transform with lambda.

Reply:
That looks more like a proper name for a utils class

Reply:
Forget it~ I just forget to remove it